pykafka/broker.py

```diff
@@ -348,3 +348,25 @@ def fetch_consumer_group_offsets(self, consumer_group, preqs):
             self.connect_offsets_channel()
         req = OffsetFetchRequest(consumer_group, partition_requests=preqs)
         return self._offsets_channel_req_handler.request(req).get(OffsetFetchResponse)
+
+    ##########################
+    #  Group Membership API  #
+    ##########################
```

Comment:
Consider renaming these methods to match the names of the requests in [the docs](https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-GroupMembershipAPI):

* `join_group`
* `leave_group`
* `sync_group`
* `heartbeat`